### ISSUE TITLE
Duda TestNoEffect-4 [E2]

### ISSUE THREAD

**Issue by vicente999** [RodrigoToroIcarte/IIC2113-2023-1#158]
Hola! Tengo una duda, que en el test 4 de TestNoEffect en esta etapa del juego:
<img width="314" alt="image" src="https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/53841116/a0f63975-ecbb-4ef0-ae6b-6a826d197bb3">
El jugador juega una carta con daño 10 dejando 0 el arsenal del rival, sin embargo en vez de terminar la partida, se regresa al menu principal
<img width="254" alt="image" src="https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/53841116/4f06d896-1387-4aa8-8b21-10d358c66c6d">
<img width="232" alt="image" src="https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/53841116/776513c0-7c5b-4ceb-9af1-a6f858d70682">

Se supone que si jugando una carta se hace daño al rival y se deja el arsenal en 0 se termina el juego al tiro

**Comment by RodrigoToroIcarte** [comment]
No, técnicamente el juego continúa hasta que el rival reciba daño teniendo cero cartas en su arsenal o hasta que se termine el turno y tenga cero cartas:

![Screenshot from 2023-09-10 19-09-10](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/7000518/4b6ad011-676f-4e1e-9404-748aa62416bc)

Es decir, si el daño hubiese sido 11 el juego termina. Como fue justo 10, el juego continúa. Pero si se termina el turno entonces pierde quien tiene cero cartas en el arsenal. Acá hay una discusión al respecto: https://github.com/RodrigoToroIcarte/IIC2113-2023-1/issues/60

**Comment by 0DASAC** [comment]
Hola, en ese mismo test Por que pierde el jugador al terminar su turno si todavia le quedan 9 cartas en el arcenal?

**Comment by RodrigoToroIcarte** [comment]
Porque al terminar el turno automáticamente pierde el jugador que tenga 0 cartas en su arsenal. En el test, pierde el HHH con cero cartas en su arsenal cuando termina el turno:

![Screenshot from 2023-09-23 11-36-25](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/7000518/8c3be10c-ca4b-4218-b964-6c1d89e2bf86)

... en este caso, sería una victoria vía _count out_.
